pkgs/development/tools/rust/cargo-cyclonedx/default.nix

```diff
@@ -0,0 +1,41 @@
+{ rustPlatform
+, fetchFromGitHub
+, lib
+, pkg-config
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-cyclonedx";
+  version = "0.3.7";
+
+  src = fetchFromGitHub {
+    owner = "CycloneDX";
+    repo = "cyclonedx-rust-cargo";
+    rev = "${pname}-${version}";
+    hash = "sha256-xr3YNjQp+XhIWIqJ1rPUyM9mbtWGExlFEj28/SB8vfE=";
+  };
```

Comment:
This source does not exist, it looks like you want the fetchCrate.

Reply:
I'm not sure I understand what you mean. From what I can see, the source does exist and it builds sucessfully. Is that not true on your local machien? Or are you saying that fetchCrate is generally preferred to fetchFromGithub for Rust packages?

Reply:
Sorry, I made a mistake. It can indeed be built successfully.